scss/bootstrap/overrides/style/_type.scss

```diff
@@ -27,8 +27,13 @@ h6, .h6 { font-weight: $crm-font-weight-h6; }
   margin: 0 0 15px 0;
 
   > h1,
-  > h2 {
+  > h2,
+  > h3,
+  > h4,
+  > h5,
+  > h6 {
     font-size: em(18);
+    font-weight: $crm-font-weight-h1;
```

Comment:
`$headings-font-weight` is a default bootstrap variable
```scss
/* default type.scss */
h1, h2, h3, h4, h5, h6,
.h1, .h2, .h3, .h4, .h5, .h6 {
  font-family: $headings-font-family;
  font-weight: $headings-font-weight;
  line-height: $headings-line-height;
  color: $headings-color;
}
```
`$crm-font-weight-*` are ours, and they are used to differentiate the different weights between headings, which as you can se in default bootstrap is always the same, but shouldn't in Shoreditch
```scss
/* our custom type.scss */
h1, .h1 { font-weight: $crm-font-weight-h1; }
h2, .h2 { font-weight: $crm-font-weight-h2; }
h3, .h3 { font-weight: $crm-font-weight-h3; }
h4, .h4 { font-weight: $crm-font-weight-h4; }
h5, .h5 { font-weight: $crm-font-weight-h5; }
h6, .h6 { font-weight: $crm-font-weight-h6; }
```

as for the "Why is it always h1": because the font-weight for any heading in the `.page-header` should always be the one of the h1, not matter what actual heading we use in `.page-header`. It should always look, visually, like it's an h1, even if you're using h6